cloud-provider-aws.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: cloud-provider-aws
+  version: 1.31.1
+  epoch: 0
+  description: The AWS cloud provider provides the interface between a Kubernetes cluster and AWS service APIs.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+    GO111MODULE: "on"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/cloud-provider-aws
+      tag: v${{package.version}}
+      expected-commit: 94cb7eefd72f6577a0f7840f61a47ebd4aed06f4
+
+subpackages:
+  - name: aws-cloud-controller-manager
+    description: The AWS Cloud Controller Manager is the controller that is primarily responsible for creating and updating AWS loadbalancers (classic and NLB) and node lifecycle management.
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/aws-cloud-controller-manager
+          output: aws-cloud-controller-manager
+          ldflags: |
+            -w -s
+            -X k8s.io/component-base/version.gitVersion=v${{package.version}}
+            -X main.gitVersion=v${{package.version}}
+    test:
+      pipeline:
+        - runs: |
+            aws-cloud-controller-manager --help
+            aws-cloud-controller-manager --version
+
+  - name: ecr-credential-provider
+    description: The credential provider is a binary that is executed by kubelet to provide credentials for images in ECR.
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/ecr-credential-provider
+          output: ecr-credential-provider
+          ldflags: |
+            -w -s
+            -X k8s.io/component-base/version.gitVersion=v${{package.version}}
+            -X main.gitVersion=v${{package.version}}
+    test:
+      pipeline:
+        - runs: |
+            ecr-credential-provider --help
+            ecr-credential-provider --version
+
+  - name: "aws-cloud-controller-manager-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream Dockerfile"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}/bin"
+          ln -sf /usr/bin/aws-cloud-controller-manager ${{targets.contextdir}}/bin/aws-cloud-controller-manager
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes/cloud-provider-aws
+    strip-prefix: v
```

Comment:
if we are switching to version streams then we also need to filter based on tags here so we'll append `tag-filter: v1.31` here for 1.31 version stream.